.github/workflows/ci.yml

```diff
@@ -33,6 +43,12 @@ jobs:
           - Code Checks
           - CloudFormation Templates Checks
           - Integration Tests Config Checks
+          - Python 3.6 AWS Batch CLI Tests
+          - Python 3.7 AWS Batch CLI Tests
+          - Python 3.8 AWS Batch CLI Tests
+          - Python 3.9 AWS Batch CLI Tests
+          - Python 3.8 AWS Batch CLI Tests Coverage
+          - Code Checks AWS Batch CLI
```

Comment:
Shall we have these as part of a separate job?

Reply:
Moved